content/agent/prometheus.md

```diff
@@ -0,0 +1,202 @@
+---
+title: Prometheus Check
+kind: documentation
+---
+
+## Overview
+
+This page details how to collect metrics and events from a Prometheus data source by writing an Prometheus Check, a Python plug-in to the Datadog Agent. This page looks first at the `PrometheusCheck` interface, and then propose a simple Prometheus Check that collects timing metrics and status events from [Kube DNS](https://github.com/DataDog/integrations-core/blob/master/kube_dns/check.py).
+
+## Setup
+
+First off, ensure you've properly installed the [Agent](http://app.datadoghq.com/account/settings#agent) on your machine. If you run into any issues during the setup, [contact our support](/help).
+
+## Prometheus check interface 
+
+`PrometheusCheck` is [a mother class](https://github.com/DataDog/dd-agent/blob/master/checks/prometheus_check.py) providing a structure and some helpers
```

Comment:
`PrometheusCheck` is [a mother class](https://github.com/DataDog/dd-agent/blob/master/checks/prometheus_check.py) providing a structure and some helpers to collect metrics, events, and service checks exposed via Prometheus. Minimal configuration for checks based on this class include:

Reply:
👍